### PR TITLE
Remove reference to deprecated default query executor

### PR DESCRIPTION
**Pull request by findinpath**
`onTrino().executeQuery` vs static `QueryExecutor.query` is confusing,
since both are the same thing.

Replace reference to `QueryExecutor.query` with
`onTrino().executeQuery`

### CODE REVIEW

**Review by kokosing — APPROVED** [review]

File: testing/trino-product-tests/src/main/resources/tempto-configuration.yaml [hunk 1]
```diff
@@ -4,8 +4,6 @@ hdfs:
     uri: http://${databases.hive.host}:50070
 
 databases:
-  default:
-    alias: presto
```
Comment (alexjo2144) [inline testing/trino-product-tests/src/main/resources/tempto-configuration.yaml:8]: Nit, get rid of the extra newline between databases and hive

**Comment by findinpath** [comment]
Still to be done is to update the contents of `io.trino.tests.product.TestCreateDropView` which depends on `io.trino.tempto.sql.SqlContexts.createViewAs`

There is an ongoing PR for coming up with an extended version for `createViewAs` method in https://github.com/trinodb/tempto/pull/88

File: pom.xml [hunk 2]
```diff
@@ -58,7 +58,7 @@
         <dep.drift.version>1.14</dep.drift.version>
         <dep.scribejava.version>6.9.0</dep.scribejava.version>
         <dep.selenium.version>3.141.59</dep.selenium.version>
-        <dep.tempto.version>186</dep.tempto.version>
```
Comment (kokosing) [inline pom.xml:61]: separate commit

**Review by kokosing — APPROVED** [review]

**Comment by kokosing** [comment]
Thank you!

**Pull request merged**